apis/apps/v1beta3/emqxenterprise_types.go

```diff
@@ -22,8 +22,12 @@ import (
 )
 
 type License struct {
-	Data       []byte `json:"data,omitempty"`
+	// Data represents the byte data of the license content
```

Comment:
Data contains the secret data. Each key must consist of alphanumeric characters, '-', '_' or '.'. The serialized form of the secret data is a base64 encoded string, representing the arbitrary (possibly non-string) data value here. Described in https://tools.ietf.org/html/rfc4648#section-4